master/usage/calicoctl.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installing and Configuring calicoctl
```

Comment:
That could work but then I feel like the page should have additional content talking about using it and probably some example commands and what not.  Which could be good for this page to have.  Though it might start getting a little long at that point and I think these usage pages should strive to be short(ish).